models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
I could see a single team sometimes having more than one github or more than one instagram, although I agree this wouldn't be super common.

I think the biggest argument _for_ this join model is that we can use these for Teams, Events, Districts etc. That will let us easily re-use the same model and templates and such with less duplicated code. If we wanted to store more complicated data about a SocialConnection (like scraping posts? last updated time? fan counts?) we'd be able to store them on these models as well.